raiden/network/protocol.py

```diff
@@ -349,17 +354,22 @@ def receive(self, data):
                 if log.isEnabledFor(logging.WARN):
                     log.warn(str(e))
                 return
+            except:
+                log.exception('unexpected exception raised.')
```

Comment:
`exception` uses [ERROR level](https://docs.python.org/2.7/library/logging.html#logging.Logger.exception), since the exception is unknown and thus not properly handled I think that is a good level. Do you want me to reduce the logging level anyways?